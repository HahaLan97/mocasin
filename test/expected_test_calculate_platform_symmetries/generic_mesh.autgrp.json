{"automorphisms": [18,[16, 11, 10, 9, 3, 5, 1, 2, 6, 7],["(1, 4)", "(2, 5)", "(2, 8)", "(3, 6)", "(5, 8)", "(6, 9)", "(6, 12)", "(7, 10)", "(7, 13)", "(9, 12)", "(10, 13)", "(11, 14)", "(16, 17)"]]}