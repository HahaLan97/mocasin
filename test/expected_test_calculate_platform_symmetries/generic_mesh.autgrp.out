Platform Graph:{'processor_0000': [('processor_0000', 41702000), ('processor_0001', 58369000), ('processor_0002', 75036000), ('processor_0003', 91702000), ('processor_0004', 58369000), ('processor_0005', 75036000), ('processor_0006', 91702000), ('processor_0007', 108370000), ('processor_0008', 75036000), ('processor_0009', 91702000), ('processor_0010', 108370000), ('processor_0011', 125040000), ('processor_0012', 91702000), ('processor_0013', 108370000), ('processor_0014', 125040000), ('processor_0015', 141700000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0001': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0002': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0003': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0004': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0005': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0006': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0007': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0008': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0009': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0010': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0011': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0012': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0013': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0014': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0015': [('processor_0000', 25022000), ('processor_0001', 25022000), ('processor_0002', 25022000), ('processor_0003', 25022000), ('processor_0004', 25022000), ('processor_0005', 25022000), ('processor_0006', 25022000), ('processor_0007', 25022000), ('processor_0008', 25022000), ('processor_0009', 25022000), ('processor_0010', 25022000), ('processor_0011', 25022000), ('processor_0012', 25022000), ('processor_0013', 25022000), ('processor_0014', 25022000), ('processor_0015', 25022000), ('processor_0016', 666670000), ('processor_0017', 666670000)], 'processor_0016': [('processor_0016', 2500000), ('processor_0017', 2500000), ('processor_0000', 666670000), ('processor_0001', 666670000), ('processor_0002', 666670000), ('processor_0003', 666670000), ('processor_0004', 666670000), ('processor_0005', 666670000), ('processor_0006', 666670000), ('processor_0007', 666670000), ('processor_0008', 666670000), ('processor_0009', 666670000), ('processor_0010', 666670000), ('processor_0011', 666670000), ('processor_0012', 666670000), ('processor_0013', 666670000), ('processor_0014', 666670000), ('processor_0015', 666670000)], 'processor_0017': [('processor_0016', 2500000), ('processor_0017', 2500000), ('processor_0000', 666670000), ('processor_0001', 666670000), ('processor_0002', 666670000), ('processor_0003', 666670000), ('processor_0004', 666670000), ('processor_0005', 666670000), ('processor_0006', 666670000), ('processor_0007', 666670000), ('processor_0008', 666670000), ('processor_0009', 666670000), ('processor_0010', 666670000), ('processor_0011', 666670000), ('processor_0012', 666670000), ('processor_0013', 666670000), ('processor_0014', 666670000), ('processor_0015', 666670000)]}Symmetry group generators:[[[16, 17]], [[11, 14]], [[10, 13]], [[7, 10]], [[9, 12]], [[6, 9]], [[3, 6]], [[5, 8]], [[2, 5]], [[1, 4]]]
Correspondence:{0: 'processor_0000', 1: 'processor_0001', 2: 'processor_0002', 3: 'processor_0003', 4: 'processor_0004', 5: 'processor_0005', 6: 'processor_0006', 7: 'processor_0007', 8: 'processor_0008', 9: 'processor_0009', 10: 'processor_0010', 11: 'processor_0011', 12: 'processor_0012', 13: 'processor_0013', 14: 'processor_0014', 15: 'processor_0015', 16: 'processor_0016', 17: 'processor_0017'}